bloom-housing/bloom
# 1348/Add photo upload to the Listing form

# Pull Request Template

## Issue

Addresses #1348 

## Description

This PR adds the ability to view, add, replace, or delete a listing photo. It also includes a way to see the new photo on the public site.

**NOTE:** there's a related issue to deal with before this can be merged: #1480 – this PR has some changes that are possibly not the best way to handle this.

## Type of change


## How Can This Be Tested/Reviewed?

You can upload an image for new and existing listings, and then verify they can be previewed or published.


## Checklist:
